### ISSUE TITLE
Options to make sure names are the same length

### ISSUE THREAD

**Issue by mastilver** [open-cli-tools/concurrently#166]
Would you accept a PR that adds a `--min-prefix-length` options?

When using wildcards command, it's harder to make sure the name are of the same size (for aesthetic reasons)

ei.

```
[babel] Successfully compiled 26 files with Babel.
[webpack] Compiled successfully.
```

Would become (using `--min-prefix-length 7`)

```
[babel  ] Successfully compiled 26 files with Babel.
[webpack] Compiled successfully.
```

**Comment by govindrai** [comment]
This would be an excellent enhancement. I have some super long service names and super short service names. Logs are super painful to traverse.

**Comment by ssukienn** [comment]
This would be such a nice feature

**Comment by gustavohenke** [comment]
yes @ssukienn!

**Comment by gustavohenke** [comment]
Duplicate of #417

**Comment by gustavohenke** [comment]
🚢 This is now available in v9.0.0!
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
